sqlglot/parser.py

```diff
@@ -1547,7 +1554,10 @@ def _parse_types(self):
                 self.raise_error("Expecting >")
 
         if type_token in self.TIMESTAMPS:
-            tz = self._match(TokenType.WITH_TIME_ZONE)
+            tz = (
+                self._match(TokenType.WITH_TIME_ZONE)
+                or type_token == TokenType.TIMESTAMPTZ
```

Comment:
was this an old bug?

Reply:
I think so yeah, timestamptz would be parsed as timestamp.